### DISCUSSION TITLE
Error when linking department

### DISCUSSION THREAD

**Discussion by AppDesv** [LiveHelperChat/livehelperchat#2047] [General]
This error occurs when trying to add a user to a department after creating the user. The same error occurred on demo.livehelperchat.com

**Comment by AppDesv** [comment]
![Captura de Tela 2024-02-07 às 17 24 58](https://github.com/LiveHelperChat/livehelperchat/assets/142036461/265bb0bd-51d9-49aa-8674-ece7b59a9a66)
![Captura de Tela 2024-02-07 às 17 25 06](https://github.com/LiveHelperChat/livehelperchat/assets/142036461/48c66318-b550-43ae-b768-3e71637d0d17)

**Reply by remdex** [reply]
On demo I fixed. It was was incorrect permission for cache/default.log file please make sure web app can write to it.
